filecheck/checker.ts

```diff
@@ -290,48 +277,100 @@ async function checkCompression(filePath: string, options: CheckerOptions) {
   }
 }
 
-function canCheckFile(filePath: string) {
-  const filePathParts = filePath.split(path.sep);
-
+function isRelevantFile(filePath: string) {
   return (
-    filePathParts.includes("files") &&
-    !filePathParts.includes("node_modules") &&
-    !/\.(DS_Store|html|json|md|txt|yml)$/i.test(filePath)
+    filePath.includes(`files${path.sep}`) &&
+    !/\.(DS_Store|ini)$/i.test(filePath)
   );
 }
 
+function isBinaryFile(filePath: string) {
```

Comment:
I'd prefer to keep the name `svg` is not a binary file an will be checked.

Reply:
I've renamed the function to `needsChecking`, as it is no longer binary only and we compress svg too.